full.yml

```diff
@@ -442,7 +442,7 @@ paths:
         '200':
           description: OK Response
           schema:
-            $ref: '#/definitions/StudentContactsResponse'
+            $ref: '#/definitions/StudentContactsByStudentResponse'
```

Comment:
nit: `s/By/For` to be consistent with operationId. Also, the name is pretty long, though I don't have any better ideas.